### ISSUE TITLE
Add support for Descriptors

### ISSUE THREAD

**Issue by Fonta1n3** [GlacierProtocol/GlacierProtocol#74]
Glacier users would greatly benefit if the protocol incorporated [descriptors](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md). The python script could display a string in the descriptor format of their multisig setup. Then the user would very easily be able to import their multisig wallet into their bitcoin core node with descriptor based `importmulti` and create a watch-only wallet that would effortlessly build unsigned transactions for the "withdrawal" process. This would also eliminate the need for the user to keep records of the redeem script and addresses, they could instead keep a paper copy of the descriptor.

This most importantly would allow Glacier to support HD Multisig in a user friendly way. I gave up on Glacier bc it does not support HD multisig, this is huge issue for privacy. If anyone would benefit from HD multisig it is exactly the kind of people who use Glacier.

**Comment by Fonta1n3** [comment]
For backwards compatibility it would just be adding a script that allows you to convert your private keys to pub keys and into a descriptor. I am not suggesting to break backwards compatibility, but the feature should be added and would not be difficult to do. Would also like to see more reliance on a node then web based block explorers like blockchain.info, utilizing descriptors would make this  more seamless.

**Comment by bitcoinhodler** [comment]
Agreed, and the descriptor can be extracted from the redeem script alone. A Glacier address can also be imported into Core using only the redeem script and the address, no descriptor needed.

There's an open issue #54 to use a full node, but that will require major changes to Glacier's process documentation, significant changes to GlacierScript, an entirely new bit of software to speak RPC to the node, and a thorough security review afterward. A descriptor is just one tiny part of that. Would it add any value by itself?
